assets/js/components/adminbar/AdminBarClicks.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Admin Bar Clicks component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_USER } from '../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as CORE_SITE } from '../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as MODULES_SEARCH_CONSOLE } from '../../modules/search-console/datastore/constants';
+import { extractSearchConsoleDashboardData } from '../../modules/search-console/util';
+import DataBlock from '../data-block';
+import AdminBarPreview from './AdminBarPreview';
+const { useSelect } = Data;
+
+const AdminBarClicks = ( { classNames } ) => {
+	const url = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+	} ) );
+	const reportArgs = {
+		...dateRangeDates,
+		dimensions: 'date',
+		url,
+	};
+	const searchConsoleData = useSelect( ( select ) => select( MODULES_SEARCH_CONSOLE ).getReport( reportArgs ) );
+
+	if ( undefined === searchConsoleData ) {
+		return <AdminBarPreview />;
+	}
+
+	const {
+		totalClicks,
+		totalClicksChange,
+	} = extractSearchConsoleDashboardData( searchConsoleData );
```

Comment:
We can't use this utility anymore since it is coupled to the format of the old return value which combines both date ranges into a single range in the returned report data.
See here for an example of how these are handled in a similar new component:
https://github.com/google/site-kit-wp/blob/6f6ea93b7971159bc506b8d6decaf0af1c4c59f2/assets/js/modules/search-console/components/dashboard/DashboardClicksWidget.js#L97-L99
Note that this (linked) component doesn't use true multi-date a range report either so the latest and older data need to be sourced a little differently from the returned report data.

Reply:
Updated @aaemnnosttv